js-ajax/js-ajax-1/dom.js

```diff
@@ -0,0 +1,21 @@
+const cards=document.getElementById('cards')
+const p=document.createElement('div') //**** this one
```

Comment:
The problem is here. You need to create new element in each iterration, so this need to be moved inside buildUI function. 
As it is now, the div you are creating here it's the only one append in the html and the content is from the last item of the response

Reply:
@atodoroski Thank you. I updated the code and added some css style.

Reply:
thank you